### PR TITLE
add notebook verbosity setting

### PR DESCRIPTION
**Pull request by meganrogge**
part of #180470

@amunger I couldn't figure out how to trigger the original aria message, could you pls test this?

### CODE REVIEW

File: src/vs/workbench/contrib/notebook/browser/notebookEditorWidget.ts [hunk 1]
```diff
@@ -2229,18 +2230,18 @@ export class NotebookEditorWidget extends Disposable implements INotebookEditorD
...
 	private _cellFocusAria(cell: ICellViewModel, focusItem: 'editor' | 'container' | 'output') {
 		const index = this._notebookViewModel?.getCellIndex(cell);
-
+		const verboseLabel = this.configurationService.getValue(AccessibilityVerbositySettingId.Notebook);
 		if (index !== undefined && index >= 0) {
 			let position = '';
 			switch (focusItem) {
 				case 'editor':
-					position = `the inner ${cell.cellKind === CellKind.Markup ? 'markdown' : 'code'} editor is focused, press escape to focus the cell container`;
+					position = `the inner ${cell.cellKind === CellKind.Markup ? 'markdown' : 'code'} editor is focused` + verboseLabel ? `, press escape to focus the cell container` : '';
```
Comment (rebornix) [inline src/vs/workbench/contrib/notebook/browser/notebookEditorWidget.ts:2238]:
```suggestion
					position = `the inner ${cell.cellKind === CellKind.Markup ? 'markdown' : 'code'} editor is focused` + (verboseLabel ? `, press escape to focus the cell container` : '');
```

File: src/vs/workbench/contrib/notebook/browser/notebookEditorWidget.ts [hunk 2]
```diff
@@ -2229,18 +2230,18 @@ export class NotebookEditorWidget extends Disposable implements INotebookEditorD
...
 					break;
 				case 'output':
-					position = `the cell output is focused, press escape to focus the cell container`;
+					position = `the cell output is focused` + verboseLabel ? `, press escape to focus the cell container` : '';
```
Comment (rebornix) [inline src/vs/workbench/contrib/notebook/browser/notebookEditorWidget.ts:2241]:
```suggestion
					position = `the cell output is focused` + (verboseLabel ? `, press escape to focus the cell container` : '');
```

File: src/vs/workbench/contrib/notebook/browser/notebookEditorWidget.ts [hunk 3]
```diff
@@ -2229,18 +2230,18 @@ export class NotebookEditorWidget extends Disposable implements INotebookEditorD
...
 					break;
 				case 'container':
-					position = `the ${cell.cellKind === CellKind.Markup ? 'markdown preview' : 'cell container'} is focused, press enter to focus the inner ${cell.cellKind === CellKind.Markup ? 'markdown' : 'code'} editor`;
+					position = `the ${cell.cellKind === CellKind.Markup ? 'markdown preview' : 'cell container'} is focused` + verboseLabel ? `, press enter to focus the inner ${cell.cellKind === CellKind.Markup ? 'markdown' : 'code'} editor` : '';
```
Comment (rebornix) [inline src/vs/workbench/contrib/notebook/browser/notebookEditorWidget.ts:2244]:
```suggestion
					position = `the ${cell.cellKind === CellKind.Markup ? 'markdown preview' : 'cell container'} is focused` + (verboseLabel ? `, press enter to focus the inner ${cell.cellKind === CellKind.Markup ? 'markdown' : 'code'} editor` : '');
```

**Review by rebornix — COMMENTED** [review]
the ternary operator has a very low precedence

**Review by bhavyaus — APPROVED** [review]

**Pull request merged**